contracts/wallet.sol

```diff
@@ -574,29 +568,16 @@ contract Wallet is ENSResolvable, AddressWhitelist, SpendLimit, GasTopUpLimit, L
         _;
     }
 
-    /// @dev This is a bulk transfer convenience function, used to migrate contracts.
-    /// @notice If any of the transfers fail, this will revert.
-    /// @param _to is the recipient's address, can't be the zero (0x0) address: transfer() will revert.
-    /// @param _assets is an array of addresses of ERC20 tokens or 0x0 for ether.
-    function bulkTransfer(address payable _to, address[] calldata _assets) external onlyOwnerOrSelf {
```

Comment:
This method is no longer used anywhere right (e.g. wallet migration)?

Reply:
We don't really use it, ut the main reason I removed it was to reduce the size, we can add it later when we merge the 3 limits.